soc/riscv/nordic_nrf/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2024 Nordic Semiconductor
```

Comment:
@carlocaione may want a signed off for 2d93018da19ae962087097ed08ed9a31994d42d9 :-)